src/main/java/org/opentripplanner/apis/gtfs/mapping/CarsAllowedMapper.java

```diff
@@ -0,0 +1,17 @@
+package org.opentripplanner.apis.gtfs.mapping;
+
+import javax.annotation.Nonnull;
+import org.opentripplanner.apis.gtfs.generated.GraphQLTypes.GraphQLCarsAllowed;
+import org.opentripplanner.transit.model.network.CarAccess;
+
+public class CarsAllowedMapper {
+
+  @Nonnull
+  public static GraphQLCarsAllowed map(@Nonnull CarAccess carsAllowed) {
```

Comment:
We don't use `@Nonnull` annotation, you can remove it.

Reply:
I also removed the `@Nonnull` annotations from the `BikesAllowedMapper`